pkg/sentry/kernel/thread_group.go

```diff
@@ -235,6 +235,11 @@ type ThreadGroup struct {
 	// execed is protected by the TaskSet mutex.
 	execed bool
 
+	// reaper indicates first threadgroup of the pid namespace. This will
```

Comment:
Suggested comment:

// reaper indicates this is the first ThreadGroup of the PIDNamespace (with PID==1). This field is immutable.

Reply:
Thanks, will update.